docs/uml-class-diagrams/middleware02/test/MiddlewareCommLinkManagerImplementationTests/MiddlewareCommLinkManagerImplementationTests.puml

```diff
@@ -2,6 +2,14 @@
 
 package edu.oakland.test.middleware02 {
     class MiddlewareCommLinkManagerImplementationTests {
+        + void rfidOffSet()
+        + void storeRequestSent()
+        + void emptyTrackData()
+        + void someTrackData()
+        + void enoughTrackData()
+        + void commDatabaseInterfaceIsNotNull()
+        + void gisManagerIsNotNull()
+        + void storedDataCheck()
```

Comment:
Just a few minor naming correction to better describe the tests, then you're all set to go!
```suggestion
        + void rfidDoesNotMatchStopsTestCase()
        + void storeRequestSent()
        + void emptyDatabaseReturnsEmptyTrackData()
        + void notEnoughDataReturnsPartialTrackData()
        + void enoughTrackDataReturnsFullData()
        + void commDatabaseInterfaceIsNotNull()
        + void gisManagerIsNotNull()
        + void storedTrackDataCheck()
```